src/pages/Subject.tsx

```diff
@@ -1,54 +1,75 @@
 import useRouteMetadataContext from '@/hooks/useRouteMetadataContext';
-import { Button, Card, CardContent, Grid, Typography } from '@mui/material';
+import {
+  Breadcrumbs,
+  Card,
+  CardContent,
+  Grid,
+  styled,
+  Typography,
+} from '@mui/material';
 import { NavLink } from 'react-router-dom';
 
+
 const Subject = () => {
   const ctx = useRouteMetadataContext();
+  
+  const CustomStyledCard = styled(Card)(({ theme }) => ({
+    height: '100%',
+    borderRadius: "20px",
+    padding: theme.spacing(3),
+    boxShadow: theme.shadows[4],
+    fontSize: "20px",
+    background: theme.palette.mode == "dark"? "rgba(104, 109, 118, 0.3)" : "rgba(238, 238, 238, 0.8)",
+    backdropFilter: "blur(10px)"
+  }));
+
+  const CustomStyledNavLink = styled(NavLink)(({theme}) => ({
+    textDecoration: 'none',
+    color: theme.palette.mode == "dark" ? "#fff" : "#6b2021",
+    transition: "color .3s ease-in-out",
+    "&:hover":{
+      boxShadow: theme.palette.mode == "dark" ? "inset 0 0 300px 0 #fff" : "inset 0 0 200px 0 #000",
+      color: theme.palette.mode == "dark"? "black": "white",
+    }
+  }))
+
+  
+
   return (
     <div>
-      <Button
-        variant="contained"
-        component={NavLink}
-        to={ctx.getPreviousPath()}>
-        Back
-      </Button>
       <Typography variant="h3">Discrete Math</Typography>
       <Typography variant="subtitle1">
-        Category: {ctx.category?.name}
+        Category:{' '}
+        <Breadcrumbs aria-label="breadcrumb">
+          <NavLink to="/">Home</NavLink>
+          <NavLink to="http://localhost:5173/category/0/subject/0">

```

Comment:
we should not hard code this path, the category id/index should be getting from the context. And always start with `/`. In production, navigate to localhost:5173 will break the app

Reply:
Changed the hard coded path and made it ctx.getSubjectPath()